custom_components/frigate/sensor.py

```diff
@@ -174,7 +175,7 @@ def name(self) -> str:
     @property
     def state(self) -> str:
         """Return the state of the sensor."""
-        return str(self.coordinator.server_status or "unknown")
+        return str(self.coordinator.server_status or STATUS_UNKNOWN)
```

Comment:
Can the "unknown" status ever actually happen? Doesn't look like it.

Reply:
True, not realistically